Options/WeakAurasOptions.lua

```diff
@@ -7126,13 +7126,21 @@ function WeakAuras.CreateFrame()
         if not(combinedText == "") then
           combinedText = combinedText.."\n\n";
         end
-        combinedText = combinedText.."--"..childId.."\n";
+
+        local formatString
```

Comment:
Well you need a string to look for when you are matching so we can just add something at the end that you can look for and keep the translation string sane